### ISSUE TITLE
Extract CNN Features of the proposed objects

### ISSUE THREAD

**Issue by praisethemoon** [qqwweee/keras-yolo3#395]
Good day,

Is there a possibility to extract object features (from the last conv layer) along with their bounding box upon prediction?

Thanks!

**Comment by OtiTantaoui** [comment]
Also interested in this question. It should be possible by normally!

**Comment by lcltopismine** [comment]
Yes you can, but you need to dig out source code and wrap some code from here.
